src/app/components/file-upload/file-upload.component.ts

```diff
@@ -27,19 +28,60 @@ export class FileUploadComponent implements OnInit {
 
   public regex = /\.([0-9a-z]+)(?=[?#])|(\.)(?:[\w]+)$/gmi;
 
+  public loadTestConfig: string;
+
+  public formData: FormData = new FormData();
+
+  public sessionStorage: Storage;
+
+  public secondsUntilETA: number;
+
+  public swaggerSummary: SwaggerSummary;
+
+  public swaggerUploadResponse: SwaggerUploadResponse;
+
   /** Used to reset the file input */
   @ViewChild('fileIn')
   fileInput: ElementRef;
 
   public swag: Swag;
 
-  constructor(private formBuilder: FormBuilder, private http: HttpClient, public router: Router) {
-  }
+  public window: Window;
```

Comment:
`window` also should not be a field of the component.

Reply:
It actually appears that `window` is not even in use within this component. However, even if it was, it would still be accessible through the global scope, which is why it should not be a field.